spring-core/src/main/java/com/baeldung/factorybean/NonSingleToolFactory.java

```diff
@@ -0,0 +1,56 @@
+package com.baeldung.factorybean;
+
+import org.springframework.beans.factory.config.AbstractFactoryBean;
+
+public class NonSingleToolFactory extends AbstractFactoryBean<Tool> {
+    private int factoryId;// standard setters and getters
```

Comment:
Please remove this comments //standard setters and getters.

Such comment should only be present in code samples in the article, but not in the source code.